### DISCUSSION TITLE
Crosswalk dataset

### DISCUSSION THREAD

**Discussion by Marnonel6** [xN1ckuz/Crosswalks-Detection-using-YOLO#1] [General]
Hi,

Well done on the project. I'm looking for a crosswalk dataset to use to train YOLOv7. Can you please link me your dataset?

Thank you.

**Comment by xN1ckuz** [comment]
> Hi,
> 
> Well done on the project. I'm looking for a crosswalk dataset to use to train YOLOv7. Can you please link me your dataset?
> 
> Thank you.

Thank you mate.
The dataset used for training is in the "training_dataset" folder. We took some images personally, others from internet. Then we augmented the dataset on Roboflow. If you want to use it just download the folder, the dataset is labeled obviously, and is divided for training and validation, as we wrote on the read me.
Feel free to ask something else if you need :)

**Reply by Marnonel6** [reply]
Thank you! Could you get YOLO to keep detecting the current objects it was trained on and your newly added crosswalk or did it only detect crosswalks?

**Reply by xN1ckuz** [reply]
I don't know if i understood your question correctly. We labeled the dataset only on crosswalks, then we used the pre-trained model and then run 300 epochs on our dataset. So with this code you can only detect crosswalks, but if you re-label the images you could easily train the model on other objects (traffic lights, road signs, etc. ). Let me know if this is the right answer for you

**Reply by Marnonel6** [reply]
Perfect thank you!

**Reply by xN1ckuz** [reply]
Feel free to ask if you need something else. Cheers!
